packages/react-router-dom/package.json

```diff
@@ -42,6 +42,7 @@
   "dependencies": {
     "@babel/runtime": "^7.12.13",
     "history": "^4.9.0",
+    "path-to-regexp": "^2.4.0",
```

Comment:
This is already part of the dependency tree as `path-to-regexp` is already [a dependency](https://github.com/ReactTraining/react-router/blob/master/packages/react-router/package.json#L44) of `react-router`, which itself is a dependency of `react-router-dom` (as you can see below).

Reply:
Correct, this isn't needed.